custom_components/tuya_local/devices/CT20W_pir_motion_detector.yaml

```diff
@@ -0,0 +1,79 @@
+name: CT20W PIR Motion Detector
+products:
+  - id: 205546373333989b6bf8
+    name: CT20W PIR Motion Detector
+  - id: ejptirvg36uwwdwu
+    name: "CT20W"
+primary_entity:
+  entity: alarm_control_panel
+  dps:
+    - id: 110
+      type: string
+      name: alarm_state
+      mapping:
+        - dps_val: disarm
+          value: disarmed
+        - dps_val: away
+          value: armed_away
+        - dps_val: stay
+          value: armed_home
+secondary_entities:
+  - entity: binary_sensor
+    name: PIR State
+    class: sound
```

Comment:
Class should be `motion` for a PIR.  If the correct class is used, the name should be not needed, as translated names will be available for the class.